libs/components/data-manager/src/lib/modules/data-manager/models/data-manager-state.ts

```diff
@@ -42,10 +72,21 @@ export class SkyDataManagerState {
     };
   }
 
+  /**
+   * Returns the `SkyDataViewState` for the specified view.
+   * @param viewId The ID for the view.
+   * @returns The `SkyDataViewState` for the specified view.
+   */
   public getViewStateById(viewId: string): SkyDataViewState {
     return this.views.find((view) => view.viewId === viewId);
   }
 
+  /**
+   * Adds a `SkyDataViewState` to a view or updates an existing one.
```

Comment:
```suggestion
   * Adds or updates the `SkyDataViewState` for the view.
```
Disregard if this isn't right, but I found myself wondering what `one` referred to and thought this was clearer. And I thought it was odd that we had `a` in front of `SkyDataViewState` and `view` here but `the` in the previous item.